sdmetrics/single_table/new_row_synthesis.py

```diff
@@ -86,6 +86,25 @@ def compute_breakdown(cls, real_data, synthetic_data, metadata=None,
         except IncomputableMetricError:
             categorical_fields = []
 
+        def update_column_name(char, column_name):
+            return column_name.replace(char, '_') if char in column_name else column_name
+
+        def update_lists(column, new_column):
+            for field_list in [numerical_fields, categorical_fields]:
+                if column in field_list:
+                    field_list.remove(column)
+                    field_list.append(new_column)
+
+        for column in real_data.columns:
+            new_column = column
+            for char in ['\n', '.', "'"]:
```

Comment:
Can we change the `update_column_name` method to take in a list of characters to replace and loop on its own? And can we rename it to `strip_characters` or something like that and just make it a utility function?